webpack.config.js

```diff
@@ -0,0 +1,174 @@
+import { CleanWebpackPlugin } from 'clean-webpack-plugin';

```

Comment:
it doesn't look like prettier has been ran on many of these files (brackets should not have spaces according to the rules)